src/instruments/src/PFD/SpeedIndicator.tsx

```diff
@@ -241,7 +256,7 @@ export const MachNumber = ({ mach }) => {
         }
     }, [showMach, machPermille]);
 
-    if (Number.isNaN(mach)) {
+    if (!mach.isNormal()) {
```

Comment:
So I actually suspect that in a lot of cases, Functional Test will be displayed. For example, I believe there is multiple test procedures where you can artificial values, e.g. a slowly descending RA value, and then can cross-check the validity by looking at the PFD. As I would assume that artificially generated values are flagged as Functional Test (FT), this check would not be accurate if we assume that a similar test exists for the Mach number. Maybe this (and a lot of other values) actually should only be hidden in the NCD and FW case.

Maybe we should check in on Discord to see if we can find some more evidence to corroborate this.